target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -227,7 +227,10 @@ ramips_setup_interfaces()
 	awm002-evb-4M|\
 	awm002-evb-8M|\
 	bdcom,wap2100-sk|\
-	c20i|\
+	c20i|)
+		ucidef_add_switch "switch0" \
+			"1:lan:3" "2:lan:4" "3:lan:1" "4:lan:2" "0:wan" "6@eth0"
+		;;
```

Comment:
If you change it this way, you will change all of the devices above.

Instead, remove the c20i here and add it to the matching case for c50 and c20-v1.

Reply:
You mean the pipes '|' separators conclude the above devices until the next semicolon ';;' set, so in my current PR it means that the following models will share the same switch config, which could end up wrong/bricked for them?
```
	asus,rt-ac57u|\
	atp-52b|\
	awm002-evb-4M|\
	awm002-evb-8M|\
	bdcom,wap2100-sk|\
```

Will try your suggestion in this PR as another commit and when all is fine I will make a total new sqaushed single PR. Still have to find out if how/if the Github Signed off is included.